core/src/main/scala/org/locationtech/rasterframes/RasterFunctions.scala

```diff
@@ -48,311 +48,328 @@ trait RasterFunctions {
+    rf_make_zeros_tile(cols, rows, cellType.name)
+
+  /** Create a column constant tiles of zero */
+  def rf_make_zeros_tile(cols: Int, rows: Int, cellTypeName: String): TypedColumn[Any, Tile] = {
+    import org.apache.spark.sql.rf.TileUDT.tileSerializer
+    val constTile = encoders.serialized_literal(F.tileZeros(cols, rows, cellTypeName))
+    withTypedAlias(s"rf_make_zeros_tile($cols, $rows, $cellTypeName)")(constTile)
+  }
+
+  /** Creates a column of tiles containing all ones */
+  def rf_make_ones_tile(cols: Int, rows: Int, cellType: CellType): TypedColumn[Any, Tile] =
+    rf_make_ones_tile(cols, rows, cellType.name)
+
+  /** Creates a column of tiles containing all ones */
+  def rf_make_ones_tile(cols: Int, rows: Int, cellTypeName: String): TypedColumn[Any, Tile] = {
+    import org.apache.spark.sql.rf.TileUDT.tileSerializer
+    val constTile = encoders.serialized_literal(F.tileOnes(cols, rows, cellTypeName))
+    withTypedAlias(s"rf_make_ones_tile($cols, $rows, $cellTypeName)")(constTile)
+  }
+
+  /** Where the rf_mask tile contains NODATA, replace values in the source tile with NODATA */
+  def rf_mask(sourceTile: Column, maskTile: Column): TypedColumn[Any, Tile] =
     Mask.MaskByDefined(sourceTile, maskTile)
 
-  /** Where the mask tile equals the mask value, replace values in the source tile with NODATA */
-  def mask_by_value(sourceTile: Column, maskTile: Column, maskValue: Column): TypedColumn[Any, Tile] =
+  /** Where the rf_mask tile equals the rf_mask value, replace values in the source tile with NODATA */
+  def rf_mask_by_value(sourceTile: Column, maskTile: Column, maskValue: Column): TypedColumn[Any, Tile] =
     Mask.MaskByValue(sourceTile, maskTile, maskValue)
 
-  /** Where the mask tile DOES NOT contain NODATA, replace values in the source tile with NODATA */
-  def inverse_mask(sourceTile: Column, maskTile: Column): TypedColumn[Any, Tile] =
+  /** Where the rf_mask tile DOES NOT contain NODATA, replace values in the source tile with NODATA */
+  def rf_inverse_mask(sourceTile: Column, maskTile: Column): TypedColumn[Any, Tile] =
     Mask.InverseMaskByDefined(sourceTile, maskTile)
 
   /** Create a tile where cells in the grid defined by cols, rows, and bounds are filled with the given value. */
-  def rasterize(geometry: Column, bounds: Column, value: Column, cols: Int, rows: Int): TypedColumn[Any, Tile] =
-    withAlias("rasterize", geometry)(
+  def rf_rasterize(geometry: Column, bounds: Column, value: Column, cols: Int, rows: Int): TypedColumn[Any, Tile] =
+    withTypedAlias("rf_rasterize", geometry)(
       udf(F.rasterize(_: Geometry, _: Geometry, _: Int, cols, rows)).apply(geometry, bounds, value)
-    ).as[Tile]
+    )
 
   /** Reproject a column of geometry from one CRS to another. */
-  def reproject_geometry(sourceGeom: Column, srcCRS: CRS, dstCRSCol: Column): TypedColumn[Any, Geometry] =
+  def rf_reproject_geometry(sourceGeom: Column, srcCRS: CRS, dstCRSCol: Column): TypedColumn[Any, Geometry] =
```

Comment:
Should we consider `rf_reproject(Raster)` vs `st_reproject(Geometry)`?

Since a reprojection operation has a specific interpretation for Raster types, I'd be hesitant to use the same function to allow automatic coercion of Raster into Geometry for the purposes of reprojecting its extent. Doesn't seem  too much to ask the user to do something like `st_reproject(rf_extent(Raster))`.

Reply:
I think you are right here. It is probably the right thing to have the API lead user to `st_reproject(rf_extent(Raster))`. 

`rf_reproject` or `rf_warp` would operate on Tiles.

Reply:
* [ ] Rename `rf_reproject_geometry` to `st_reproject(Geometry)`
* [ ] Create a ticket for implementing `rf_reproject(Raster)`

Reply:
https://github.com/locationtech/rasterframes/issues/115

Reply:
7dccb8eb09f41b2ebc6407f5f0d3e03a767394b1